src/azure-cli/azure/cli/command_modules/acr/task.py

```diff
@@ -401,56 +400,52 @@ def acr_task_update(cmd,  # pylint: disable=too-many-locals, too-many-statements
 
     step = task.step
     branch = None
+
+    # If no context is given, use the existing context.
+    # If given context is /dev/null, set it to None.
+    # Otherwise, use the given context.
     if context_path is None:
         context_path = step.context_path
+    elif context_path.lower() == ACR_NULL_CONTEXT:
+        context_path = None
     else:
         branch = _get_branch_name(context_path)
 
+    encoded_task_content = None
+    if context_path is None:
+        # When cmd_value is given or file is given,
+        # regenerate encoded_task_content
+        if cmd_value is not None or file is not None:
+            yaml_template = get_yaml_template(
+                cmd_value, timeout, file)
+            import base64
+            encoded_task_content = base64.b64encode(
+                yaml_template.encode()).decode()
+        # When only timeout is given,
+        # encoded_task_content cannot be patched
+        elif timeout is not None:
+            raise CLIError(
```

Comment:
Could you please use the specific error type `IncorrectUsageError` instead of `CLIError`?

Reply:
done